DevCycle/DVCClient.swift

```diff
@@ -44,6 +45,7 @@ public class DVCClient {
         
         if let options = self.options {
             Log.level = options.logLevel
+            self.flushEventsInterval = Double(self.options?.flushEventsIntervalMs ?? self.defaultFlushInterval) / 1000.0
```

Comment:
Should the flush timer not be on the Event Queue class?

Reply:
Event Queue is not a public class, this way all user specified options live at the same level and are all within the DVCClient